.github/workflows/_publish-mobile-github-release.yml

```
@@ -243,13 +252,22 @@ jobs:
           fi
 
       - name: Make GitHub release latest and non-pre-release
-        if: steps.check_version.outputs.version_released == 'true'
+        if: steps.check_version.outputs.version_released == 'true' && inputs.dry_run != 'true'
```

If using `inputs.dry_run` this should just be `true` since it's a boolean, but if you use `env._DRY_RUN` then I think it would be `'true'` since it becomes a string when set in the env.
```suggestion
        if: steps.check_version.outputs.version_released == 'true' && inputs.dry_run != true
```